maps/groundbase/groundbase_areas.dm

```diff
@@ -358,3 +358,25 @@
 	icon_state = "orablacir"
 /area/groundbase/mining/explored
 	icon_state = "blublacir"
+
+// Exclude some more areas from the atmos leak event since its outside.
+/datum/event/atmos_leak/setup()
```

Comment:
you're probably going to want to do the level2 areas also, since it looks like you missed them

Reply:
Ah, I was certain that I did add them. I can correct it in a bit.